### PR TITLE
Andrew R's Project Update and Report (11/30)

### PR DESCRIPTION
**Pull request by areiter18**

### CODE REVIEW

**Review by areiter18 — COMMENTED** [review]
tweaked formatting

**Review by areiter18 — COMMENTED** [review]
tweaked formatting again

**Review by MishaCienfuegos — CHANGES_REQUESTED** [review]
Because I helped another student troubleshoot this recently, I know that the in order for your posts to show up in your People page, the file title needs to begin `YYYY-MM-DD-`. Yours has an underscore `_`, not a hyphen (`-`) at the end, which will cause issues. Would you mind switching that?

File: _posts/areiter18/2021-11-30_ProjectUpdate_StandUpReport.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    print("3.) Expenditures")
+    print("4.) Surplus (if a positive number) or Deficit(if a negative number)")
+    print("and 5.) Surplus as a Percentage of GDP.")
+    print("***************************")
+    print("All amounts of money are in thousands of United States Dollars(USD). ")
+    print("***************************")
+    print("If you see a negative number for the 3rd value it means that particular country has a budget deficit. A positive number means they have a budget surplus.")
+ ```
+
+<p>Essentially, I forgot to include the phrase ```break``` at the end, causing the code to continually loop and ask the same thing repeatedly if you selected a particular input. This was an important reminder to remove yourself from the project (I had not noticed this before) and then return to closely edit line-by-line. </p>
```
Comment (MishaCienfuegos) [inline _posts/areiter18/2021-11-30_ProjectUpdate_StandUpReport.md]: Ooh, interesting choice to use html tags here!

**Review by areiter18 — COMMENTED** [review]
tweaked file name

**Review by MishaCienfuegos — APPROVED** [review]
Thanks for editing that! Merging now.

**Pull request merged**